amazon-s3-quickstart/README.md

```diff
@@ -0,0 +1,99 @@
+# Quarkus demo: AWS S3 Client
+
+This example showcases how to use the AWS S3 client with Quarkus. As a prerequisite install Install [AWS Command line interface](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).
+
+# S3 local instance
+
+Just run it as follows:
+`docker run --rm -p 8008:4572 -e SERVICES=s3 -e START_WEB=0 -d localstack/localstack`
+
+S3 listens on `localhost:8008` for REST endpoints.
+
+Create an AWS profile for your local instance using AWS CLI:
+
+```
+$ aws configure --profile localstack
+AWS Access Key ID [None]: test-key
+AWS Secret Access Key [None]: test-secret
+Default region name [None]: us-east-1
+Default output format [None]:
+```
+
+## Create bucket
+
+Create a S3 bucket using AWS CLI and the localstack profile.
+`aws s3 mb s3://quarkus.s3.quickstart --profile localstack --endpoint-url=http://localhost:8008`
+
+# Run the demo in dev mode
+
+- Run `./mvnw clean package` and then `java -jar ./target/amazon-s3-quickstart-1.0-SNAPSHOT-runner.jar`
+- In dev mode `./mvnw clean quarkus:dev`
+
+Go to `http://localhost:8080/s3.html`, it should show a simple App to manage files on your Bucket. 
+You can upload files to the bucket via the form.
+
+Alternatively, go to `http://localhost:8080/async-s3.html` with the simple App communicating with Async resources.
+
+# Running in native
+
+You can compile the application into a native executable using:
+
+`./mvnw clean package -Pnative`
+
+and run with:
+
+`./target/amazon-s3-quickstart-1.0-SNAPSHOT-runner` 
+
+
+# Running native in container (using localstack)
+
+Build a native image in container by running:
+
+`./mvnw package -Pnative -Dnative-image.docker-build=true`
+
+Build a docker image:
+`docker build -f src/main/docker/Dockerfile.native -t quarkus/amazon-s3-quickstart .`
+
+Create a network that connect your container with localstack
+`docker network create localstack`
+
+Start localstack and connect to the network
+`docker run --rm --network=localstack --name localstack -p 8008:4572 -e SERVICES=s3 -e START_WEB=0 -d localstack/localstack`
+
+Don't forget to create a S3 bucket (see above). And run quickstart container connected to that network (note that we're using internal port of the S3 localstack)
```

Comment:
Can I suggest literally to copy the mb command instead of saying "Don't forget"?  Easier if you are following the sequence.

Reply:
Make sense